### PR TITLE
Make the open specification consistent with Microsoft.Subscription.Ad…

### PR DESCRIPTION
**Pull request by yusonMicrosoft**
…min operation API.

* Add ResourceType 'checkIdentityHealth' to Microsoft.Subscription.Admin.

* Add ResourceType 'locations/operationsStatus' to Microsoft.Subscription.Admin.

* Add ResourceType 'manifests' to Microsoft.Subscription.Admin.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by tjprescott** [comment]
@yusonMicrosoft please address the CI failures:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=24451

File: specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Subscriptions.json [hunk 1]
```diff
@@ -474,6 +504,240 @@
...
+                "resource": {
+                    "description": "The resource identifier. This should be the application identifier of the resource application.",
+                    "type": "string"
+                },
+                "client": {
+                    "description": "The client identifier. This should be the application identifier of the client application.",
+                    "type": "string"
+                },
+                "roleId": {
+                    "description": "The role identifier. This should be the ID of the APPROLE exposed by the resource application.",
```
Comment (nschonni) [inline specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Subscriptions.json]:
```suggestion
                    "description": "The role identifier. This should be the ID of the AppRole exposed by the resource application.",
```

File: specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Manifest.json [hunk 2]
```diff
@@ -0,0 +1,909 @@
+{
```
Comment (nschonni) [inline specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Manifest.json]: The file isn't referenced in the README, so Avocado is failing

File: specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/examples/Location/GetOperationsStatus.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            "body": {
+                "id": {
+                    "location": "",
+                    "name": "",
+                    "subscriptionId": ""
+                },
+                "status": "",
+                "startTime": "2017-08-01T00:00:00.00Z",
+                "endTime": "2017-08-01T00:00:00.00Z",
+                "percentComplete": "",
```
Comment (nschonni) [inline specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/examples/Location/GetOperationsStatus.json]: Type says number
```suggestion
                "percentComplete": 0,
```

File: specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Subscriptions.json (hunk 1 above)
Comment (yusonMicrosoft) [reply specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Subscriptions.json]: Thanks for you comments. Done.

File: specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/examples/Location/GetOperationsStatus.json (hunk 3 above)
Comment (yusonMicrosoft) [reply specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/examples/Location/GetOperationsStatus.json]: Done

File: specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Manifest.json (hunk 2 above)
Comment (yusonMicrosoft) [reply specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Manifest.json]: Done

**Comment by yusonMicrosoft** [comment]
Hi Nick,

Thanks for your comment in the PR. I have addressed all of them but seems there is one more error in the PR. Can you help investigate it please?
https://github.com/Azure/azure-rest-api-specs/pull/5820

Best,
Yuchen

**Comment by yusonMicrosoft** [comment]
@nschonni

**Pull request closed without merging**